src/Tickets/Emails/Email/Ticket.php

```diff
@@ -318,28 +318,36 @@ public function get_preview_context( $args = [] ): array {
 	}
 
 	/**
-	 * Get email content.
-	 *
-	 * @since 5.5.10
+	 * Get default template context for email.
 	 *
-	 * @param array $args The arguments.
+	 * @since TBD
 	 *
-	 * @return string The email content.
+	 * @return array $args The default arguments
 	 */
-	public function get_content( $args = [] ): string {
-		// @todo: Parse args, etc.
-		$is_preview = ! empty( $args['is_preview'] ) ? tribe_is_truthy( $args['is_preview'] ) : false;
-
-		// @todo @juanfra @codingmusician: we need to see if we initialize tickets from a method.
+	public function get_default_template_context(): array {
 		$defaults = [
 			'title'              => $this->get_title(),
 			'heading'            => $this->get_heading(),
-			'tickets'            => ! empty( $args['tickets'] ) ? $args['tickets'] : $this->__get( 'tickets' ),
 			'post_id'            => $this->__get( 'post_id' ),
+			'tickets'            => ! empty( $args['tickets'] ) ? $args['tickets'] : $this->__get( 'tickets' ),
 			'additional_content' => $this->format_string( tribe_get_option( $this->get_option_key( 'add-content' ), '' ) ),
 		];
 
-		$args = wp_parse_args( $args, $defaults );
+		return $defaults;
+	}
+
+	/**
+	 * Get email content.
+	 *
+	 * @since 5.5.10
```

Comment:
Does this need any updated `@since`?

Reply:
No, it's just git moving the code from one place to the other, [check this line](https://github.com/the-events-calendar/event-tickets/pull/2595/files#diff-cf56629cac0fe51b928ad17db72ac1591007e2d093484b7d2ff1cc7f1301f5c5L329).